doc/source/installation.rst

```diff
@@ -126,10 +126,13 @@ the ``-d`` or ``--dir=`` argument.
 6. Include in application
 .........................
 
-To include ``pyramid_oereb`` into your existing Pyramid application, you first have to include the plugin in
-your application's main method.
+In your ``setup.py`` you shouldn't directly depends on ``pyramid_oereb`` because you will not depends on any
```

Comment:
I don't understand the second part of the sentence.

Reply:
... what don't you understand?

Reply:
the "because you will not depends on any..." part.

Reply:
Can we inverse the sentence?

Reply:
Fixed :-)